src/current/v23.2/molt-fetch.md

```diff
@@ -33,7 +33,7 @@ For previous binaries, see the [MOLT version manifest](https://molt.cockroachdb.
 
 Complete the following items before using MOLT Fetch:
 
-- Follow the recommendations in [Best practices](#best-practices).
+- Follow the recommendations in [Best practices](#best-practices) and [Security](#security).
```

Comment:
It might be good to call the Security section Security Recommendations here and below.